Dockerfile

```diff
@@ -8,12 +8,12 @@ RUN apt-get update -y \
     && pecl install ev \
     && rm -rf /var/lib/apt/lists/*
 
-RUN docker-php-ext-install mbstring gd zip \
+RUN docker-php-ext-install mbstring gd pdo_mysql mysqli zip \
     && docker-php-ext-enable gd zip
 
 RUN a2enmod rewrite \
     && curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
 
 COPY . .
 
-CMD php -S 0.0.0.0:8000 -t public
+CMD php -S 0.0.0.0:8000
```

Comment:
@LeinadAlberto  To initialize this Dockerfile you need to start public with the following command from the Dockerfile "CMD php -S 0.0.0.0:8000 -t public"